src/styles/FormFooter.css.js

```diff
@@ -0,0 +1,13 @@
+import styled from "react-emotion";
+
+// The bottom padding here is related to the body padding-bottom in App.css to stack sticky FormFooter and App Navigation Footer
+const StyledFooter = styled("div")`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  padding: 5px 0 50px 0;
+  background-color: #e6e6e6;
+  width: 100%;
```

Comment:
what if you added a box shadow here to give it more of a 3D effect?
`box-shadow: 0px 0px 13px #212529;`

Ex:
![Screen Shot 2019-07-23 at 4 16 43 PM](https://user-images.githubusercontent.com/5697474/61748184-47d5b200-ad65-11e9-83e9-56e5ac54fe36.png)

Reply:
Nice! I like that a lot so I updated before merging. 👍